flink-rpc/flink-rpc-akka/src/main/java/org/apache/flink/runtime/rpc/akka/AkkaUtils.java

```diff
@@ -121,7 +121,7 @@ private static String getLogLevel() {
             return "INFO";
         }
         if (LOG.isWarnEnabled()) {
-            return "WARN";
+            return "WARNING";
```

Comment:
I am actually not sure whether `TRACE` is supported. According to https://doc.akka.io/docs/akka/current/general/configuration.html#custom-application-conf it is not.

Reply:
True, lets fix that while we're at it.